### PR TITLE
UI: Fix hypervisor not selected by default when deploying VM from ISO

### PR DESCRIPTION
**Pull request by nvazquez**
### Description

This PR fixes the default value for the hypervisor dropdown at the Deploy VM wizard when ISO is selected

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [x] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [x] Minor
- [ ] Trivial

### CODE REVIEW

**Review by shwstppr — APPROVED** [review]
Checked with QA server, hypervisor is sent in the request

File: ui/src/views/compute/DeployVM.vue [hunk 1]
```diff
@@ -1974,7 +1975,9 @@ export default {
             this.options[name] = response
 
             if (name === 'hypervisors') {
-              this.hypervisor = response[0] && response[0].name ? response[0].name : null
+              const hw = response[0] && response[0].name ? response[0].name : null
```
Comment (sureshanaparti) [inline ui/src/views/compute/DeployVM.vue:1978]:
```suggestion
              const hypervisorFromResponse = response[0] && response[0].name ? response[0].name : null
```

**Review by slavkap — APPROVED** [review]
LGTM
manually tested

File: ui/src/views/compute/DeployVM.vue (hunk 1 above)
Comment (nvazquez) [reply ui/src/views/compute/DeployVM.vue]: Done thanks

**Pull request merged**